lib/tag-generator.js

```diff
@@ -43,34 +43,37 @@ export default class TagGenerator {
     }
 
     switch (this.scopeName) {
-      case 'source.c':        return 'C';
-      case 'source.cpp':      return 'C++';
-      case 'source.clojure':  return 'Lisp';
-      case 'source.capnp':    return 'Capnp';
-      case 'source.coffee':   return 'CoffeeScript';
-      case 'source.css':      return 'Css';
-      case 'source.css.less': return 'Css';
-      case 'source.css.scss': return 'Css';
-      case 'source.elixir':   return 'Elixir';
-      case 'source.fountain': return 'Fountain';
-      case 'source.gfm':      return 'Markdown';
-      case 'source.go':       return 'Go';
-      case 'source.java':     return 'Java';
-      case 'source.js':       return 'JavaScript';
-      case 'source.js.jsx':   return 'JavaScript';
-      case 'source.jsx':      return 'JavaScript';
-      case 'source.json':     return 'Json';
-      case 'source.julia':    return 'Julia';
-      case 'source.makefile': return 'Make';
-      case 'source.objc':     return 'C';
-      case 'source.objcpp':   return 'C++';
-      case 'source.python':   return 'Python';
-      case 'source.ruby':     return 'Ruby';
-      case 'source.sass':     return 'Sass';
-      case 'source.yaml':     return 'Yaml';
-      case 'text.html':       return 'Html';
-      case 'text.html.php':   return 'Php';
-      case 'text.tex.latex':  return 'Latex';
+      case 'source.c':                 return 'C';
+      case 'source.cpp':               return 'C++';
+      case 'source.clojure':           return 'Lisp';
+      case 'source.capnp':             return 'Capnp';
+      case 'source.cfscript':          return 'ColdFusion';
+      case 'source.cfscript.embedded': return 'ColdFusion';
```

Comment:
The embedded scope is needed in case function declarations occur within a `cfm` template.  The CFML scripting language allows you to embedd script-like functions in tag markup, thus the need for searching for symbols within embedded scripts in a `cfm` template. 

Example:

**MyTemplate.cfm**
```coldfusion
<cfscript>
function helper(){}
function anotherHelper(){}
</cfscript>

<cfoutput>
Markup goes here that uses defined helpers above
</cfoutput>
```

As you can see from the sample, CFML templates allow you to generate script blocks where functions can be declared.